docs/users/welcome.md

```diff
@@ -22,3 +22,35 @@ What to do (and not to do) in the community.
 - Yuck Someone Else's Yum, as they say.
   - If you have serious concerns, do bring them up, either privately or on the appropriate Board. Please remember the "do" rules, and be mindful that the present fandom climate might make some people twitchier about certain subjects.
 - Continue pressing an issue after someone has asked you to stop. In threads, **OP has final saying on what is allowed on their thread**. Making a different thread to discuss something OP would rather not touch upon is allowed, but try to avoid "subtle" jabs at the previous OP (trust us, they're never subtle).
+
+## FAQS
+
+### Can I necropost*?
+
+On BobaBoard, you can necro everything you want.
+
+*a.k.a. answer to older threads that haven't seen activity in a while
+
+### I really need a post deleted!
+
+Just write under the post that you'd like for that to happen, and I'll do so after confirmation. If I don't notice the request, please bump it up in !bobaland, which I check most often. 
+
+If you see a deletion request in the wild, feel free to *privately* bring it up to me on Discord. You don't need to tell me if it's yours 😉
+
+### Is BobaBoard a [Insert Social Network] Clone?
+
+BobaBoard isn't an anything clone. Even beyond the concept of anonymity-first, I strive to rethink our assumptions on the shape of online fandom spaces.
+
+Inspiration for the platform will come from many different sources: my Peak Fandom Fun was on Tumblr; I often come back to Amino Apps and Discord Servers as tools for the creation of smaller, "independent" hubs of fandom activity; Reddit and Quora offer insights on less-identity-centered spaces of discussion; And (I swear I'm not making this up) I've even considered lifting features from LinkedIn.
+
+There are many ways BobaBoard could evolve as it grows, yet it's important to not be prescriptive when it comes to product development. Building a platform is, in a way, like raising a child: you can try to steer them in a certain direction, but it's important to leave them space to breathe and find their own path.
+
+### **Will there be Complex Content Filters and All That Jazz?**
```

Comment:
Mostly same as above. The way I'd change is is I'd keep the content as you've written it out but structure it in a way that makes it easier to see how things work at a glance:
@: does this
+: does that